big-practice/src/js/pages/dashboard.page.js

```diff
@@ -193,7 +220,7 @@ class Dashboard {
   /**
    * Handle creating new movie when user clicks create movie button
    */
-  handleModalFormCreateBtn() {
+  createNewMovie() {
```

Comment:
Main target of this function should be add an event for create button, not create new movie

Reply:
Fixed